web/backend/routes/routes.ts

```diff
@@ -44,8 +49,15 @@ const matches = [
   },
 ];
 
-router.get("/api/v1/matches", (context) => {
-  context.response.body = matches;
-});
+router
+  .get("/", (ctx) => {
+    ctx.response.body = "This is the home route";
+  })
+  .post("/add-participant", createParticipant)
+  .post("/add-matches", createMatches)
+  .post("/get-participants", findAll)
```

Comment:
.post("/get"...) not really logical

try to use api/v1/ before each endpoint to make it coherent

also refactor so you have
 participants/
participants/{id}

please follow https://restfulapi.net/resource-naming/

Reply:
ok